content/docs/2.15/reference/_index.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "Reference"
+weight = 2
++++
+
+Reference information for the KEDA autoscaler.
+
+- [Scaled Object specification](./scaledobject-spec)
+- [ScaledJob specification](./scaledjob-spec.md)
+- [Events]
```

Comment:
```suggestion
- [ScaledJob specification](./scaledjob-spec)
- [Kubernetes Events](./events)
```

Reply:
@tomkerkhove @nate-double-u @thisisobate I thought there were still merge issues due to failed checks. Have those been resolved? 

I'm good with the content. I think we've resolved most of the review questions you had. If we merge now, any corrections we have to make will be minor.